src/components/Core/style.scss

```diff
@@ -2,7 +2,21 @@
 @tailwind components;
 @tailwind utilities;
 
-@import url('https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200');
+@font-face {
+  font-family: 'Material Symbols Rounded';
+  font-style: normal;
+  font-weight: 400;
+  src: url('./fonts/icons.woff2') format('woff2');
```

Comment:
A separate issue is that this will help only a little - the font is still a network resource downloaded when used, it will just download without additional delay from CDN. The proper solution is to preload the font before it is used, however from what I quickly searched, this should be done by modifying the <link href="font.woff" /> element in the index page, so it might be the case that it can be solved only on project side

Reply:
yes, that's why I chose this solution.